packages/wallet-service/webpack.config.js

```diff
@@ -24,7 +24,9 @@ module.exports = {
     filename: '[name].js',
   },
   target: 'node',
-  externals: [nodeExternals()],
+  externals: [nodeExternals({
+    allowlist: [new RegExp("@wallet-service/common*")],
```

Comment:
Why do you need this?

Reply:
Yarn workspaces hoists modules to the root node_modules directory leaving any incompatible (different semver, etc.) modules with the dependent workspace's node_modules directory, which is the case of our internal package

So we set it as a [external](https://webpack.js.org/configuration/externals/), which is loaded during runtime from the node_modules and not included in the webpack bundle

Reply:
The bundle gets too big if we allow webpack to bundle all dependencies so we remove them from the bundle (they [get loaded in runtime](https://webpack.js.org/configuration/externals/)

But we can't do this with the common module, otherwise it would not be seen by the serverless-monorepo package